airbyte-integrations/connectors/source-xero/source_xero/manifest.yaml

```diff
@@ -0,0 +1,2842 @@
+version: "0.44.0"
```

Comment:
This is extremely outdated. Let's make sure that you re-import connector into Builder with running custom components locally. Happy to show how to run them.

Reply:
Updated and tested with connector builder :white_check_mark:, 
Apart from custom extractor, if it's DpathExtractor, There are no problems, we could use the manifest directly to generate UI in builder
![image](https://github.com/airbytehq/airbyte/assets/47392334/7d9de6cb-5385-4506-9372-50118070ee70)